src/vl-mapactions-draw-action.js

```diff
@@ -32,8 +32,20 @@ export class VlDrawAction extends VlMapAction {
     });
     const drawInteraction = new Draw(options);
     interactions.push(drawInteraction);
-    if (options.snapping) {
-      interactions.push(new VlSnapInteraction(options.snapping.layer || layer));
+    if (options.snapping !== undefined) {
```

Comment:
Dus `options.snapping` false is niet meer toegelaten?

Reply:
Jawel, verderop wordt er nog gechecked of het een boolean is, en enkel indien die true is snapping op de 'teken-layer' aangezet.

Reply:
Zie ook unit test.